book/src/configuration.md

```diff
@@ -45,6 +45,7 @@ hidden = false
 | `auto-info` | Whether to display infoboxes | `true` |
 | `true-color` | Set to `true` to override automatic detection of terminal truecolor support in the event of a false negative. | `false` |
 | `rulers` | List of column positions at which to display the rulers. Can be overridden by language specific `rulers` in `languages.toml` file. | `[]` |
+| `max_line_len` | Set the maximum line length for text wrapping | `79` |
```

Comment:
This ends up being `max-line-length` in configuration and when doing `:set max-line-length 80` because of the kebab-case rename:

 https://github.com/helix-editor/helix/blob/76d55c0d9e12b7ae9f08d2b722c44473c9b0ef2a/helix-view/src/editor.rs#L108